wmi_client_wrapper/wrapper.py

```diff
@@ -56,9 +57,10 @@ def _make_credential_args(self):
 
         # the format is user%pass
         # NOTE: this is an injection vulnerability
-        userpass = "--user={username}%{password}".format(
+        userpass = "--user={domain}{username}%{password}".format(
             username=self.username,
             password=self.password,
+            domain = str(self.domain) + "/" if self.domain else "" 
```

Comment:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/5491690/186418758-a83ab877-a33c-42cf-8a45-380f3245efcf.png">

Just as an FYI.

I'll run https://github.com/psf/black over the master before I deploy the new release to PyPI so it's all good.